locales/English/gamecreate.php

```diff
@@ -66,15 +66,52 @@
 					</p>
 				</div>
 			</div>
-			<select class = "gameCreate" name="newGame[phaseMinutes]">
+			<select class = "gameCreate" name="newGame[phaseMinutes]" id="selectPhaseMinutes">
 			<?php
 				$phaseList = array(5,7, 10, 15, 20, 30, 60, 120, 240, 360, 480, 600, 720, 840, 960, 1080, 1200, 1320,
 					1440, 1440+60, 2160, 2880, 2880+60*2, 4320, 5760, 7200, 8640, 10080, 14400);
 
 				foreach ($phaseList as $i) { print '<option value="'.$i.'"'.($i==1440 ? ' selected' : '').'>'.libTime::timeLengthText($i*60).'</option>'; }
 			?>
 			</select>
+			
+			<p id="phaseSwitchPeriodPara">
+				<strong>Time Until Phase Swap</strong></br>
+				<select class = "gameCreate" id="selectPhaseSwitchPeriod" name="newGame[phaseSwitchPeriod]">
+				<?php
+				$phaseList = array(-1, 10, 15, 20, 30, 60, 120, 240, 360, 480, 600, 720, 840, 960, 1080, 1200, 1320,

```

Comment:
I think there's 2 ways this can be improved. 1 I think is needed, the other is just a nice to have. 

1. (needed) only give offerings from 10 minutes - 6 hours max, and have every hour be an option after an hour, so like 
-1, 10,15,20,30,60,120,180,240,300,360. Anything above that seems too long for the live game scene. 
2. (ideal but not needed) adjust offerings allowed based on the phase length, so don't allow a shorter time to switch than the phase. Cause if the phase length is 30 minutes, but it switches after 10 minutes then it doesn't make sense.

Reply:
I did 1, but haven't found time for 2. Might find that time in future.

Reply:
Okay gotcha, I'll probably do part 2 before I merge it into the code base. Thanks for doing 1.

Reply:
I ended up doing #2, it's pushed.